README.md

```diff
@@ -130,80 +146,96 @@ ti.tidbMapDatabase ("tpch_test")
 
 |    Key    | Default Value | Description |
 | ---------- | --- | --- |
-| spark.tispark.pd.addresses |  127.0.0.1:2379 | PD Cluster Addresses, split by comma |
-| spark.tispark.grpc.framesize |  268435456 | Max frame size of GRPC response |
-| spark.tispark.grpc.timeout_in_sec |  10 | GRPC timeout time in seconds |
-| spark.tispark.meta.reload_period_in_sec |  60 | Metastore reload period in seconds |
-| spark.tispark.plan.allow_agg_pushdown |  true | If allow aggregation pushdown (in case of busy TiKV nodes) |
-| spark.tispark.plan.allow_index_read |  true | If allow index read (which might cause heavy pressure on TiKV) |
-| spark.tispark.index.scan_batch_size |  20000 | How many row key in batch for concurrent index scan |
-| spark.tispark.index.scan_concurrency |  5 | Maximal threads for index scan retrieving row keys (shared among tasks inside each JVM) |
-| spark.tispark.table.scan_concurrency |  512 | Maximal threads for table scan (shared among tasks inside each JVM) |
-| spark.tispark.request.command.priority |  "Low" | "Low", "Normal", "High" which impacts resource to get in TiKV. Low is recommended for not disturbing OLTP workload |
-| spark.tispark.coprocess.streaming |  false | Whether to use streaming for response fetching (Experimental) |
-| spark.tispark.plan.unsupported_pushdown_exprs |  "" | A comma-separated list of expressions. In case you have a very old version of TiKV, you might disable some of the expression push-down if not supported |
-| spark.tispark.plan.downgrade.index_threshold | 1000000000 | If index scan ranges on one region exceed this limit in the original request, downgrade this region's request to table scan rather than original planned index scan, by default the downgrade is turned off |
-| spark.tispark.show_rowid |  false | If to show implicit row Id if exists |
-| spark.tispark.db_prefix |  "" | A string indicating the extra database prefix for all databases in TiDB to distinguish them from Hive databases with the same name |
-| spark.tispark.request.isolation.level |  "SI" | Isolation level means whether do the resolve lock for the underlying tidb clusters. When you use the "RC", you will get the newest version of record smaller than your tso and ignore the locks. And if you use "SI", you will resolve the locks and get the records according to whether the resolved lock is committed or aborted  |
-
-## Log4j Configuration
+| `spark.tispark.pd.addresses` |  `127.0.0.1:2379` | The addresses of PD cluster, which are split by comma |
+| `spark.tispark.grpc.framesize` |  `268435456` | The maximum frame size of gRPC response |
+| `spark.tispark.grpc.timeout_in_sec` |  `10` | The gRPC timeout time in seconds |
+| `spark.tispark.meta.reload_period_in_sec` |  `60` | The reloading period of metastore in seconds |
+| `spark.tispark.plan.allow_agg_pushdown` |  `true` | If allow aggregation pushdown (in case of busy TiKV nodes) |
```

Comment:
What does it mean by "If allow aggregation pushdown"?

Reply:
should be whether `aggregations are allowed to push down to TiKV`